docs/lesson-plans/week-1.md

```diff
@@ -102,42 +95,541 @@ every week may not end up helping you as much as it would for our project, and y
+[C4C Jumpstart Docs](https://learn.c4cneu.com) (where this should be)
+
+[C4C Slack](https://c4cneu.slack.com/)
+
+[C4C GitHub Repository](https://github.com/Code-4-Community)
+
+[C4C Website](https://c4cneu.com)
+
+[C4C Dev Docs](https://docs.c4cneu.com)
+
+#### Making edits to the dictionary
+
+If there's a term you'd like to see added to the dictionary, updated, or formatted differently, you
+can click on the pencil icon in the upper right to make an edit (you can do this to the script pages
+too if there's a change you want there instead). You will probably need an account, 
+and then it should take you to an in-browser edit page. Once you're ready to make the change,
+you should be able to go to the bottom to leave a quick message detailing what you changed
+(you don't have to do the extended description). Then you'll be able to create a branch, and we'll
+review it to make sure it's a good edit, and let you merge it in. Then you'll have successfully 
+contributed to the C4C dictionary.
+
+## Project Setup
+
+Now we'll be setting up your project. If you want to see videos for setting up the 
+frontend and backend, see below.
+
+The overall setup process will just involve cloning the repos on your computer and 
+then reading the READMEs to see how to set up each individual project.
+
+![frontend-setup-video]()
+
+![backend-setup-video]()
+
+### Installing IntelliJ
+
+The way that you'll be interacting with a lot of the code for this project will be through IntelliJ.
+Go to the [install link](https://www.jetbrains.com/idea/download/) and download the
+'Ultimate' edition. Since you're a student, you aren't limited to the 30-day trial that would otherwise
+be applied.
+
+Once you've downloaded an installed it (you'll probably need to create an account with your student 
+email), open it so that the welcome page shows up.
+
+>Note: some people prefer to use text editors like [Microsoft's VSCode](https://code.visualstudio.com/) or
+>[GitHub's Atom](https://atom.io/) instead of IntelliJ for working on the frontend. We'll be focusing
+>on using IntelliJ for all of this, but feel free to try those out if you're interested.
+
+### Cloning the repositories with Git
+
+Now that you've gotten IntelliJ set up, we'll get to cloning the repositories on your computer.
+
+>Note: you may need a GitHub account to do this. If you encounter an error trying to clone it,
+>then then just sign up and and retry the steps after logging into GitHub in IntelliJ.
+
+Inside of IntelliJ, go to File > New... > Project From Version Control..., make sure the 
+`Version Control` box is set to `Git`, paste in the following links, and choose where you 
+want them to be downloaded to. 
+
+The links you'll want to use are 
+- Backend: https://github.com/Code-4-Community/jumpstart-backend/tree/starter-code
+- Frontend: __*GET STARTER CODE TAG DONE*__
+
+Once you've done that, you're ready to install other programs we'll need and build the projects.
+
```

Comment:
Now that we've gotten to actually installing/running stuff, we should have notes reminding readers to reach out to us if they have issues -- they're bound to run into unexpected errors.

Reply:
second this

Reply:
Done